src/seedsigner/views/scan_views.py

```diff
@@ -4,6 +4,7 @@
 from embit.descriptor import Descriptor
 
 from seedsigner.gui.screens.screen import RET_CODE__BACK_BUTTON
+from seedsigner.gui.screens import scan_screens
```

Comment:
Noting that this additional import adds a considerable initial-delay moving from MainMenu into Seeds.